### PR TITLE
Update 49

### PR DESCRIPTION
**Pull request by stringhandler**
~~DNM - Waiting for binaries~~

Ready to merge

### CODE REVIEW

**Review by philipr-za — APPROVED** [review]
lgtm

File: _updates/2021-03-10-update-49.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+author: Mike Berry
+thumbnail: update-background.jpg
+title: Tari Release v0.8.6
+subtitle:
+class: subpage
+---
+
+## Tari Release v0.8.6
+
+This release contains a fix for nodes running with a pruning horizon ([#2677](https://github.com/tari-project/tari/pull/2677)), as well as the final separation of wallet from the base node.
```
Comment (hansieodendaal) [inline _updates/2021-03-10-update-49.md:14]: **NIT:**

Maybe "... as well as the final separation of **the** wallet **and miner** from the base node ..."

**Review by hansieodendaal — APPROVED** [review]
LGTM

**Review by CjS77 — APPROVED** [review]
+1

**Pull request merged**